### DISCUSSION TITLE
What's the podman equivalent of Docker's `volume-opt` parameter in mount strings?

### DISCUSSION THREAD

**Discussion by dstaley** [containers/podman#13378] [Q&A]
I'm trying to translate the following `--mount` string from Docker to podman:

```
"src=services,dst=/services,volume-opt=device=:/volume3/Services,\"volume-opt=o=addr=ganymede.localdomain,vers=3,soft\",type=volume,volume-driver=local,volume-opt=type=nfs"
```

When running a container with this mount string, I get the error:

```
Error: volume-opt: invalid mount option
```

I understand that the options passed to `--mount` are basically parsed out and passed to the OS's underlying `mount` command, so I assume that `volume-opt` isn't getting translated to whatever it needs to be for `mount` to understand it.

I've seen a few sample commands that create volumes using `--opt`, however I don't see anything like that in the [docs](https://docs.podman.io/en/latest/markdown/podman-run.1.html#mount-type-type-type-specific-option) for the `--mount` argument.

**Comment by rhatdan** [comment]
This looks like an issue, if Docker supports this and we don't.

**Reply by mheon** [reply]
Concur, this looks like a feature request. @dstaley Can you open an RFE for this?

**Comment by rhatdan** [comment]
I already created an issue out of this.
